src/hooks/useIssuesData.tsx

```diff
@@ -1,16 +1,18 @@
 import { useState } from "react";
 import { useIssuesContext } from "contexts/IssuesContext";
 
+import { getGithubAccountInfo } from "../utils/getGithubAccountInfo";
+
 export const useIssuesData = () => {
   const [isLoading, setIsLoading] = useState(false);
   const { setIssuesData, issuesData } = useIssuesContext();
   const fetchIssues = async (url: string) => {
     setIsLoading(true);
-    const githubAccountName = url.split("/")[3];
-    const githubRepositoryName = url.split("/")[4];
+    const accountName = getGithubAccountInfo(url).accountName;
```

Comment:
in this case you're calling `getGithubAccountInfo` twice which is not practical. Use object destructuring